services/users.js

```diff
@@ -29,6 +31,37 @@ async function getShoutouts(userId) {
   };
 }
 
+async function getUserInfo(userId) {
+  const user = await users.readBySlackId(userId);
+  const org = await organizations.read(null, user.org_id);
+
+  if (!user || !user.slack_mem_id) {
+    return {
+      statusCode: 404,
+      data: {
+        message: 'User not found',
+      },
+    };
+  }
+
+  const result = await slackUsers.getUser(userId, org.slack_org_id, org.access_token);
+  if (!result) {
+    return {
+      statusCode: 404,
+      data: {
+        message: "Couldn't get user info",
```

Comment:
This message suggests that the user may exist, but the user's info could not be fetched at the moment. This may contradict the `not found` status code of `404`, you may consider changing the message text to something like; `User does not exist`.

Reply:
I have changed this to `User not found` instead. Hopefully, this makes more sense